### PR TITLE
simple-tiles 0.6.0

### PR DESCRIPTION
**Pull request by dunn**

### CODE REVIEW

File: Library/Formula/simple-tiles.rb [hunk 1]
```diff
@@ -1,19 +1,37 @@
...
+    prefix.install "build/test"
+  end
+
+  test do
+    cp prefix/"test/api", testpath
+    resources.each do |r|
+      (testpath/"data").install r
+    end
+    system testpath/"api"
+    File.exist? "out.png"
```
Comment (xu-cheng) [inline Library/Formula/simple-tiles.rb]: Any chance to create a test without external resources?

Comment (mikemcquaid) [inline Library/Formula/simple-tiles.rb]: :+1:

Comment (DomT4) [inline Library/Formula/simple-tiles.rb]: Any update on this @dunn?

Comment (dunn) [inline Library/Formula/simple-tiles.rb]: I haven't had time to figure out a self-contained test, sorry!

Comment (DomT4) [inline Library/Formula/simple-tiles.rb]: Would the example under `Overview` on [the website](https://propublica.github.io/simple-tiles/) work?

Comment (dunn) [inline Library/Formula/simple-tiles.rb]: Wouldn't it still need one of the sample data packages for this line `simplet_vector_layer_t *layer = simplet_map_add_vector_layer(map, "PG:dbname=simple_tiles_roads");`?

Comment (DomT4) [inline Library/Formula/simple-tiles.rb]: Ah, yes, looks like it :cry:

Comment (xu-cheng) [inline Library/Formula/simple-tiles.rb]: We only need to link to the library. So this should be enough:

```c
#include <simple-tiles/simple_tiles.h>

int main(){
  simplet_map_t *map = simplet_map_new();
  simplet_map_free(map);
  return 0;
}
```

Comment (dunn) [inline Library/Formula/simple-tiles.rb]: Thanks @xu-cheng

File: Library/Formula/simple-tiles.rb [hunk 2]
```diff
@@ -14,6 +12,27 @@ class SimpleTiles < Formula
 
   def install
     system "./configure", "--prefix=#{prefix}"
-    system "make install"
+    system "make", "install"
+    prefix.install "build/test"
```
Comment (DomT4) [inline Library/Formula/simple-tiles.rb]: Would this be better placed in `share` or `libexec`?

Comment (dunn) [inline Library/Formula/simple-tiles.rb]: It's not necessary anymore since I fixed the test.  Maybe just remove it?

Comment (DomT4) [inline Library/Formula/simple-tiles.rb]: :+1:

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/f4e9ebb71785327f9c48dc5655173fdb701bf71e. Thanks @dunn! :cocktail:

**Pull request closed without merging**